apps/twitch-bot/internal/command/helpers/helpers.go

```diff
@@ -61,3 +69,18 @@ func isBroadcaster(badgeTags string) bool {
 func isModerator(badgeTags string) bool {
 	return strings.Contains(badgeTags, "moderator")
 }
+
+func checkModsCanManageCmds(context context.Context, twitchChannelId string, server *server.SenchabotAPIServer) (bool, error) {
+	configData, err := server.GetTwitchBotConfig(context, twitchChannelId, "mods_manage_cmds_enabled")
```

Comment:
// FIXME server shouldn't be used here, replace it with client version of this call.